### PR TITLE
Strip the leading SDL_ from the pixel format name

### PR DESCRIPTION
**Pull request by MyreMylar**
Fixes issue #1741.

I'm not sure this is the best way to strip 4 characters from a C string, so please let me know.

### CODE REVIEW

**Review by ankith26 — CHANGES_REQUESTED** [review]
A one-line change to achieve this could be passing `pixel_format_name + 4` directly instead of the copying to new memory. This of course, assumes that `pixel_format_name` is a non-null pointer and the string has atleast 4 chars (which should be checked, either way)

**Comment by ankith26** [comment]
```c
if (!strncmp(pixel_format_name, "SDL_", 4)) {
    pixel_format_name += 4
}
```
(untested code)
This should do maybe?

**Comment by zoldalma999** [comment]
The vidinfo type also has a getter, defined a bit above the str function. This currently does not even return a value for pixel format, so I think this could be an opportunity to fix that (and of course, strip "SDL_" off there as well).

**Comment by ankith26** [comment]
> ps: [SDL_GetPixelFormatName will always return a string starting with "SDL_"](https://github.com/libsdl-org/SDL/blob/SDL2/src/video/SDL_pixels.c#L88C13-L88C35)

Yeah I realised that too as I made my comment, but then it's safer (more future-proof) this way and does no harm.

**Comment by MyreMylar** [comment]
> The vidinfo type also has a getter, defined a bit above the str function. This currently does not even return a value for pixel format, so I think this could be an opportunity to fix that (and of course, strip "SDL_" off there as well).

Should work now:

```
import pygame

pygame.init()
print(pygame.display.Info().pixel_format)
```

Output:
```
pygame-ce 2.4.0.dev1 (SDL 2.26.5, Python 3.11.1)
PIXELFORMAT_RGB888
```

**Review by zoldalma999 — APPROVED** [review]
Looks good to me!

File: buildconfig/stubs/pygame/display.pyi [hunk 1]
```diff
@@ -29,6 +29,7 @@ class _VidInfo:
     blit_sw_A: int
     current_h: int
     current_w: int
+    pixel_format: str
```
Comment (ankith26) [inline buildconfig/stubs/pygame/display.pyi:32]: This change needs to be documented too

**Comment by MyreMylar** [comment]
Documented the `pixel_format` attribute and fixed that attribute documentation block while I was in there. Before it was formatted as python code and had an unnecessarily messy horizontal scrollbar.

👍

**Review by ankith26 — APPROVED** [review]
LGTM, thanks for the PR :tada:

**Pull request merged**